### DISCUSSION TITLE
does not work with ubuntu 24.4

### DISCUSSION THREAD

**Discussion by mr-brune** [v2rayA/v2rayA#1399] [General]
i installed it with apt method but when i set system proxy it doesn't start

**Comment by tzr0125** [comment]
looks like it need to start, using `systemctl start v2raya`.
But I still cant connect because of error: geoip.dat or geosite.dat file does not exists.

**Reply by dream2333** [reply]
same

**Reply by eronez** [reply]
If you can access github
try https://github.com/XTLS/Xray-install
if not, download .dat file manually and put it to /usr/local/share/xray.
